.travis.yml

```diff
@@ -12,6 +12,16 @@ matrix:
         - python: "3.5"
         - python: "3.6"
         - python: "3.7-dev"
+        - env: BUILD_DOCKER=1
+          servces:
```

Comment:
Is that typo here important?

Reply:
That's weird. That typo should actually prevent everything intended in this commit. But it still works.

Thanks for mentioning.